app/Entirety/entities/views.py

```diff
@@ -81,21 +84,34 @@ def get_context_data(self, **kwargs):
 
     def post(self, request, *args, **kwargs):
         selected = self.request.POST.getlist("selection")
-        if not selected:
-            messages.warning(
-                self.request,
-                "Please select an entity from the table to edit or delete.",
-            )
-            return redirect("projects:entities:list", project_id=self.project.uuid)
+        
         if self.request.POST.get("Edit"):
+            if not selected:
+                messages.warning(
+                    self.request,
+                    "Please select an entity from the table to edit.",
+                )
+                return redirect("projects:entities:list", project_id=self.project.uuid)
+            
             # TODO: error if more than one selected for edit
```

Comment:
Is this TODO still active?

Reply:
I am not sure. I checkt it and if you select multiple entites for edit just one entity is randomly selceted. So the app continues working, but still nor error is raised, so i guess this TODO is still active. However this belongs to the "enties" app and does not concern the semantics app. So maybe this is somthing for a new issue?